.github/workflows/smokeAndComponents.yml

```
@@ -0,0 +1,232 @@
+                    component/test_acl_isolatednetwork
+                    component/test_acl_isolatednetwork_delete
+                    component/test_acl_listsnapshot",
+                  "component/test_acl_listvm
+                    component/test_acl_listvolume",
+                  "component/test_acl_sharednetwork
+                    component/test_acl_sharednetwork_deployVM-impersonation
+                    component/test_user_private_gateway
+                    component/test_user_shared_network",
+                  "component/test_affinity_groups_projects
+                    component/test_allocation_states
+                    component/test_assign_vm",
+                  "component/test_concurrent_snapshots_limit
+                    component/test_cpu_domain_limits
+                    component/test_cpu_limits
+                    component/test_cpu_max_limits
+                    component/test_cpu_project_limits
+                    component/test_deploy_vm_userdata_multi_nic",
+                  "component/test_egress_fw_rules
+                    component/test_invalid_gw_nm
+                    component/test_ip_reservation",
+                  "component/test_lb_secondary_ip
+                    component/test_list_nics
+                    component/test_list_pod
+                    component/test_memory_limits",
+                  "component/test_mm_domain_limits
+                    component/test_mm_max_limits
+                    component/test_mm_project_limits
+                    component/test_network_offering
+                    component/test_non_contiguous_vlan",
+                  "component/test_persistent_networks
+                    component/test_project_configs
+                    component/test_project_limits
+                    component/test_project_resources",
+                  "component/test_project_usage
+                    component/test_protocol_number_security_group
+                    component/test_public_ip
+                    component/test_resource_limits",
+                  "component/test_regions_accounts
+                    component/test_routers
+                    component/test_snapshots
+                    component/test_stopped_vm
+                    component/test_tags
+                    component/test_templates
+                    component/test_updateResourceCount
+                    component/test_update_vm",
+                  "component/test_volumes
+                    component/test_vpc
+                    component/test_vpc_distributed_routing_offering
+                    component/test_vpc_network",
+                  "component/test_vpc_network_lbrules
+                    component/test_vpc_offerings
+                    component/test_vpc_routers
+                    component/test_vpn_users" ]
+
+    steps:
+      - name: 'Set up Python ${{ matrix.python-version }}'
+        uses: actions/setup-python@v2
```

```suggestion
        uses: actions/setup-python@v4
```

https://github.com/actions/setup-python